tests/build_certs

```diff
@@ -10,52 +24,31 @@ function _readlink() { (
 CGO_LDFLAGS_ALLOW='.*'
 QED="go run $GOPATH/src/github.com/bbva/qed/main.go"
 
-pub=$(_readlink /var/tmp/certs)
+pub=$(_readlink /var/tmp)
 tdir=$(mktemp -d /tmp/qed_build.XXX)
-mkdir -p $pub
+
 sign_path=${pub}
-cert_path=${pub}/server.crt
-key_path=${pub}/server.key
+cert_path=${pub}
 
 (
 cd ${tdir}
 
+if [ ! -f ${node_path} ]; then (
```

Comment:
I don't understand why node-exporter is here, in "build_certs" script.

Reply:
Fixed! It was a copy & paste typo